decidim/decidim
# Refactor participatory space user roles

#### :tophat: What? Why?
This PR aims to remove the duplicate code in the decidim participatory spaces, in order to ensure a consistent way of managing the user roles. 

#### Testing
1. Apply the patch
2. Visit the admin of Participatory Processes , Assemblies and Conferences 
3. Aim to perform various actions regarding the User Roes. 
4. You should not have any issues managing the users in the participatory process. 
 

:hearts: Thank you!
